src/System.Linq.Queryable/tests/System.Linq.Queryable.Tests.csproj

```diff
@@ -55,6 +55,27 @@
     <Compile Include="UnionTests.cs" />
     <Compile Include="WhereTests.cs" />
     <Compile Include="ZipTests.cs" />
+    <Compile Include="$(CommonTestPath)\System\Linq\SkipTakeData.cs">
+      <Link>Common\System\Linq\SkipTakeData.cs</Link>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
+    <Compile Include="SkipLastTests.netcoreapp.cs" />
+    <Compile Include="TakeLastTests.netcoreapp.cs" />
+  </ItemGroup>
+  <ItemGroup>
```

Comment:
The project references aren't needed any longer. Please rebase/merge on top of change https://github.com/dotnet/corefx/commit/2847e0ec9f7c9ac42d44d2bc8c6fd39e504b7a9b which I think will get your ref configurations fixed up.